src/mdserver/main/CMakeLists.txt

```diff
@@ -111,9 +115,9 @@ TARGET_LINK_LIBRARIES(mdserver
     visit_verdict
     vtkCommonCore
     vtkIOCore
-    vtktiff
-    vtkpng
-    vtkjpeg
+    ${vtktiff_LIBRARIES}
```

Comment:
I ran into this doing a spack build and if I recall, the spack package for VTK doesn't build the VTK image libraries. It relies on spack built versions. So that's why I switched to the CMake variables.

Reply:
Sounds good.  Does our image processing suffer because of this? Or are we/vtk using system versions of the libraries internally?

Reply:
My assumption has been that the vtk and system versions of the libraries were equivalent, but I don't really know. I assumed that they grabbed a snapshot of the system libraries and that they are pretty static, so any changes wouldn't be significant.